src/libsystemd/sd-netlink/netlink-util.c

```diff
@@ -178,3 +179,60 @@ int rtnl_log_parse_error(int r) {
 int rtnl_log_create_error(int r) {
         return log_error_errno(r, "Failed to create netlink message: %m");
 }
+
+void rtattr_append_attribute_internal(struct rtattr *rta, unsigned short type, const void *data, size_t data_length) {
+        size_t padding_length;
+        char *padding;
```

Comment:
doesn't matter much, but to me "char" indicates this was actually character data... I'd always use "uint8_t" for this, since these are "arbitrary padding bytes"